central/graphql/resolvers/image_components.go

```diff
@@ -77,6 +78,11 @@ type ImageComponentResolver interface {
 	TopImageVulnerability(ctx context.Context) (ImageVulnerabilityResolver, error)
 	UnusedVarSink(ctx context.Context, args RawQuery) *int32
 	Version(ctx context.Context) string
+
+	// deprecated functions
+
+	FixedIn(ctx context.Context) string
+	ID(ctx context.Context) graphql.ID
```

Comment:
No need to deprecate `ID`, can be directly removed as nothing changes for the GraphQL
schema. Request to `id` field in GraphQL query will be automatically routed to `Id` function.